setup_wizard/texture_import_setup/texture_importer_types.py

```diff
@@ -777,17 +838,18 @@ def import_textures(self, directory):
                 elif self.is_texture_identifiers_in_texture_name(['Face_ExpressionMap'], file):
                     self.set_face_expression_texture(face_material, img)
 
-                elif self.is_texture_identifiers_in_texture_name(['Weapon', 'Color'], file):
+                elif self.is_texture_identifiers_in_texture_name(['Weapon', 'Color'], file) and \
+                    not self.is_texture_identifiers_in_texture_name(['Screen'], file):  # Pela, Silverwolf
                     for weapon_material in weapon_materials:
                         if weapon_material:
-                            self.set_diffuse_texture(TextureType.BODY, weapon_material, img)
+                            self.set_diffuse_texture(TextureType.WEAPON, weapon_material, img)
 
                 elif self.is_texture_identifiers_in_texture_name(['Weapon', 'LightMap'], file) or \
-                    self.is_texture_identifiers_in_texture_name(['Weapon', 'LigthMap'], file):  # Yes, intentional typo
+                    self.is_texture_identifiers_in_texture_name(['Weapon', 'LigthMap'], file):  # Yes, intentional typo (Asta)
 
                     for weapon_material in weapon_materials:
                         if weapon_material:
-                            self.set_lightmap_texture(TextureType.BODY, weapon_material, img)
+                            self.set_lightmap_texture(TextureType.WEAPON, weapon_material, img)
```

Comment:
Note: These changes (same reason for `set_diffuse_texture()` above) to `TextureType.WEAPON` is not for fixing a bug, but rather that the `TextureType` was not being used.

Nya222 shader had the same name for each diffuse/lightmap node, but StellarToon has a `{TextureType}_Color_UV0`.